component/appcat_sla_reporter.jsonnet

```diff
@@ -76,7 +75,10 @@ local ObjectStorage = kube._Object('appcat.vshn.io/v1', 'ObjectBucket', 'appcat-
   },
 };
 
+local promRulePostgresSLA = common.PromRuleSLA(params.services.vshn.postgres.sla, 'postgres');
```

Comment:
I'd actually move this to `vshn_postgres.jsonnet`.

Reasoning:
We can enable and disable services individually. So it might be possible that we have a cluster where only redis is enabled, then we would not need the postgresql sla.
Also having everything that can belongs to a specific service in the same place makes it easier to handle IMHO.

Reply:
Actually I set it first in postgres but then I realised that maybe put them together in the same place would be easier to figure out the SLAs. But your argument is good enough and will put it back in postgres jsonnet